packages/ckeditor5-link/tests/linkimageediting.js

```diff
@@ -177,22 +200,22 @@ describe( 'LinkImageEditing', () => {
 			describe( 'a > img', () => {
 				it( 'should convert a link in an image figure', () => {
 					editor.setData(
-						'<a href="http://ckeditor.com"><img src="/assets/sample.png" alt="alt text" /></a>'
+						'<figure class="image"><a href="http://ckeditor.com"><img src="/assets/sample.png" alt="alt text" /></a></figure>'
```

Comment:
Those tests should use only `ImageBlockEditing` until we add support for linking inline images. Please avoid changing input data.